src/components/MainLayout/MobileHeader.tsx

```diff
@@ -155,6 +156,20 @@ export const MobileHeader = ({
                   </Link>
                 )}
               </Box>
+              <Suspense fallback={null}>
+                {isFeatureEnabled('vaults') && (
```

Comment:
@yahiamarzouk I think we should follow the logic from the [main header](https://github.com/coordinape/coordinape/blob/main/src/components/MainLayout/MainHeader.tsx#L39) and only show this button if there are claims, not always:

```suggestion
          {isFeatureEnabled('vaults') && showClaimsButton && (
```